src/nupic/regions/SPRegion.py

```diff
@@ -756,6 +756,10 @@ def getSpec(cls):
     return spec
 
 
+  def getAlgorithm(self):
```

Comment:
I think we should have one for the CLA Classifier, and KNNClassifier as well.  And how about RecordSensor?